### PR TITLE
Cleanup element tables

### PR DESCRIPTION
**Pull request by dericed**

### CODE REVIEW

**Comment by dericed** [comment]
https://github.com/Matroska-Org/matroska-specification/pull/29

File: index.md [hunk 1]
```diff
@@ -14,298 +34,27 @@ A more refined and detailed version of the EBML specifications is being [worked
...
-| <a name="SignatureElements"></a>SignatureElements | 1 | [7E][5B] | - | - | - | - | <abbr title="Master Elements">m</abbr> | Contains elements that will be used to compute the signature. |
-| <a name="SignatureElementList"></a>SignatureElementList | 2 | [7E][7B] | - | mult. | - | - | <abbr title="Master Elements">m</abbr> | A list consists of a number of consecutive elements that represent one case where data is used in signature. Ex: _Cluster|Block|BlockAdditional_ means that the BlockAdditional of all Blocks in all Clusters is used for encryption. |
-| <a name="SignedElement"></a>SignedElement | 3 | [65][32] | - | mult. | - | - | <abbr title="Binary">b</abbr> | An element ID whose data will be used to compute the signature. |
-
-The default values defined for the EBML header correspond to the values for a Matroska stream/file. When parsing the EBML header the default values are different, irrespective of the DocType defined.
-
-*   EBMLMaxIDLength is 4: IDs in the EBML header cannot be longer than 4 octets.
-*   EBMLMaxSizeLength is 4: Length of IDs in the EBML header cannot be longer than 4 octets.
```
Comment (dericed) [inline index.md]: I think this line of the spec was incorrect, should be 8 or less, corrected in https://github.com/Matroska-Org/matroska-specification/pull/30/commits/1a7943bf0072a6a82fb7b9fd97803e2d60c9f4ed

Comment (mbunkus) [inline index.md]: You're right.

**Comment by mbunkus** [comment]
Reviewing this is a bit hard on top of all the other commits. Let's finish and merge #29, then re-base this one on the result, and the diff should be much easier to review.

**Comment by dericed** [comment]
https://github.com/Matroska-Org/matroska-specification/pull/29 should be ready.

File: diagram.md [hunk 2]
```diff
@@ -22,7 +22,11 @@ The first picture is a simple representation of a Matroska file.
...
 
+<<<<<<< HEAD
+The [Metaseek]({{site.baseurl}}/index.html#MetaSeekInformation) section contains an index of where all of the other groups are in the file arelocated, such as the Track information, Chapters, Tags, Cues, Attachments, andso on. This element isn't technicaly REQUIRED, but you would have to searchthe entire file to find all of the other Level 1 elements if you did not haveit. This is because any of the items can occur in any order. For instance youcould have the chapters section in the middle of the Clusters. This is partof the flexibility of EBML and Matroska.
+=======
 The [MetaSeek]({{site.baseurl}}/index.html#MetaSeekInformation) section contains an index of where all of the other groups are in the file arelocated, such as the Track information, Chapters, Tags, Cues, Attachments, and so on. This element isn't technicaly REQUIRED, but you would have to search the entire file to find all of the other Level 1 elements if you did not have it. This is because any of the items can occur in any order. For instance you could have the chapters section in the middle of the Clusters. This is part of the flexibility of EBML and Matroska.
```
Comment (mbunkus) [inline diagram.md]: Here be dragons, I mean, merge conflicts.

**Pull request merged**